src/pages/login/index.tsx

```diff
@@ -37,6 +40,10 @@ const Login = () => {
     );
   };
 
+  useEffect(() => {
+    if (code) setIsSettingToken(true);
+  }, [code]);
+
   useEffect(() => {
     if (code) {
```

Comment:
```suggestion
  useEffect(() => {
    if (code) {
    	setIsSettingToken(true);
```
하나의 useEffect로 처리해도 될 것 같아요

Reply:
흠 그런가요 저는 `setIsSettingToken`는 어차피 비동기로 동작하니 어디에 위치하든 상관없다고 생각해서 같이 두어도 괜찮을 것 같아용

Reply:
어 혹시 조금 더 자세히 설명해주실 수 있으신가요?

Reply:
저도 진호님 의견처럼 하나의 useEffect로 관리하는 것이 좋아보입니다!
code가 있는 경우 동작을 한번에 이해하기 더 쉬운 것 같아요

Reply:
그런데 code가 있는 경우 isSettingToken을 true로 설정해주는데 사실 불필요한 상태 같습니다!

code 유무로만 판단해도 동일하게 동작하지 않나요?
```
{code && <div className={cx('setting-token')} />}
```

Reply:
맞네요 불필요한 상태로 보입니다 오늘 밤에 수정해서 올리겠습니다